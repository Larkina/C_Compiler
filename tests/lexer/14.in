char* c = "bad const;
printf("%s", *c);