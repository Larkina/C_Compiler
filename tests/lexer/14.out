Token{value=char, text=char, type=KEY_WORD}
Token{value=*, text=*, type=MUL}
Token{value=c, text=c, type=VAR}
Token{value==, text==, type=ASSIGN}
Incorrect token on 1 line, 6 pos: Uunclosed string const
